en/docs/guides/identity-lifecycles/lock-account.md

```diff
@@ -1,234 +1,138 @@
-			and
-			Permissions]({{base_path}}/guides/identity-lifecycles/manage-roles-overview/)
-
-			 Although the **Internal/system** role is configured by default in
-			WSO2 Identity Server, you can delete the role if necessary. To allow
-			users with the **Internal/system** role to bypass account locking,
-			you need to ensure that the role exists in WSO2 Identity Server.
-         
-         
-6.  To enable account locking for other tenants, log out and repeat the
-   steps given above from [step 2](#lockingaspecificuseraccount)
-   onwards.
-    
-## Enable claims
-
-1.  Navigate to **Main** > **Identity** > **Claims** > **List** and select the `http://wso2.org/claims` claim dialect.  
-    For more information about claims, see [Configure Claims]({{base_path}}/guides/dialects/configure-claims/).
-2.  Select the **Account Locked** claim and click **Edit**.  
-    ![account-locked-claim]({{base_path}}/assets/img/guides/account-locked-claim.png) 
-3.  Select the **Supported by Default** check box and click **Update**.
-
-    This is done to make the "Account Locked" status appear in the
-    user's profile.  
-    ![locked-status]({{base_path}}/assets/img/guides/locked-status.png)
-   
-4.  Navigate to **Main** > **Identity** > **Users and Roles** > **List** > **Users** and click on **User Profile** of the user you want to lock.
-5.  If it is the first time this particular account is being locked, a
-    text box will appear in front of the **Account Locked** field as
-    seen below.  
-    To lock the account, type true in the text box and click **Update**.
-
-    ![admin-lock-account]({{base_path}}/assets/img/guides/admin-lock-account.png)
-
-    !!! note
-        If it is not the first time you are locking this user account, there
-      will be a check box instead of the text box (as shown above) in front of
-      the **Account Locked** field. Select the check box to lock the account
-      or deselect it to unlock the account and click **Update**.
-
-## Configure the email sender
-
-[Enable the email sending configurations]({{base_path}}/deploy/configure-email-sending) of the WSO2 Identity Server.
+# Configuring Admin-Initiated Account Locking
 
-## Try it out
+WSO2 Identity Server enables privileged users to temporarily lock suspicious user accounts and prevent the users from logging in. These locked user accounts can only be unlocked by privileged users. 
+
+## Scenario
+
+**Pickup** is a cab company that has many employees who use different credentials to sign in to different internal enterprise applications. While **Sam** is an administrator at Pickup, **Larry** is a new recruit. Due to suspicious user activity, Sam wants to lock Larry's account.  
+
+Let's learn how Sam can lock Larry's user account!
+
+### Prerequisites
+- You need to [add a user]({{base_path}}/guides/identity-lifecycles/admin-creation-workflow/) named `Larry` with [login permissions]. Do not assign any roles to this user.
```

Comment:
```suggestion
- You need to [add a user]({{base_path}}/guides/identity-lifecycles/admin-creation-workflow/) named `Larry` with login permissions. Do not assign any roles to this user.
```

Reply:
Why do we say `Do not assign any roles to this user`? We've asked to assign login permissions.